src/main/java/minijava/ir/DefaultNodeVisitor.java

```diff
@@ -0,0 +1,170 @@
+package minijava.ir;
+
+import firm.nodes.*;
+
+public interface DefaultNodeVisitor extends NodeVisitor {
```

Comment:
Yeah, that's why it's good to have `NodeVisitor.Default`. I can see why you want an interface there (mixin inheritance >>> abstract classes), but it hardly justifies not using what's already there and used on master in the meantime. Also, in the cases where you use `DefaultNodeVisitor`, you *could* use an abstract class instead, e.g. there are no other base classes.